terraform-docs.sh

```diff
@@ -7,7 +7,7 @@ command -v terraform-docs > /dev/null 2>&1 || ( echo "terraform-docs not availab
 if [[ "$(terraform version | head -1)" =~ 0\.12 ]]; then
     TMP_FILE="$(mktemp /tmp/terraform-docs-XXXXXXXXXX)"
     # shellcheck disable=SC2035
-    awk -f scripts/terraform-docs.awk *.tf > "${TMP_FILE}"
+    awk -f terraform-docs.awk *.tf > "${TMP_FILE}"
```

Comment:
Are we sure this is working, with the new Makefile target that uses BIN_DIR to retrieve this file? I feel like this script will need to detect its own working dir instead... https://stackoverflow.com/questions/59895/get-the-source-directory-of-a-bash-script-from-within-the-script-itself/246128#246128

Reply:
updated